tests/sample/ext/inc/sample_ext_program_info.h

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+/**
+ * @brief Sample eBPF Extension program types.
+ */
+
+#include <stdint.h>
+
+#include "ebpf_platform.h"
+
+#include "sample_ext_helpers.h"
+
+static ebpf_context_descriptor_t _sample_ebpf_context_descriptor = {
+    sizeof(sample_program_context_t),
+    EBPF_OFFSET_OF(sample_program_context_t, data_start),
+    EBPF_OFFSET_OF(sample_program_context_t, data_end),
+    -1};
+
+// Test Extension Helper function prototype descriptors.
+static ebpf_helper_function_prototype_t _sample_ebpf_extension_helper_function_prototype[] = {
+    {EBPF_MAX_GENERAL_HELPER_FUNCTION + 1,
```

Comment:
I'd recommend having a #define somewhere for the start of the sample extension's helper functions, e.g.,
`#define SAMPLE_EXT_HELPER_FUNCTION_START EBPF_MAX_GENERAL_HELPER_FUNCTION`
and then using SAMPLE_EXT_HELPER_FUNCTION_START instead of EBPF_MAX_GENERAL_HELPER_FUNCTION here and below, which allows easy relocation of numbers.  Since this is a sample, we should make it easy for people to change ranges.